### PR TITLE
fix: update node-sass to support node 13

### LINKED ISSUE #414: Cannot install nes.css with latest node versions
**Describe the bug**
```
❯ yarn add nes.css@latest
yarn add v1.22.4
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
error nes.css@2.3.0: The engine "node" is incompatible with this module. Expected version "10.x". Got "13.11.0"
error Found incompatible module.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

**To Reproduce**
1. Install node > 13
2. Try to install nes.css via yarn

**Expected behavior**
The installation works.

**Environment:**
 - OS: MacOS Mojave
 - Build environment (i.e. NodeJS): Node 13

**Suggestion(s) for fixing this issue**
Well, I manage to install nes.css with downgrading my node version to 10.19.0. However, this is not a fix, but a workaround.

Looks like the solution is here: https://github.com/nostalgic-css/NES.css/issues/392#issuecomment-569440614

I can create a PR is you want.

### PR DESCRIPTION
**Pull request by doliG**
close #414 

**Description**
Simple update of node-sass to support node-13 installation, according to [node-sass doc](https://github.com/sass/node-sass#node-sass).

NodeJS | Minimum node-sass version | Node Module
-- | -- | --
Node 13 | 4.13+ | 79
Node 12 | 4.12+ | 72

**Compatibility**
As long as there is no note of breaking changes in 4.13, it should be compatible.

**Caveats**
:warning: I tried to check if it's working by running `npm run build` then `npm pack` and finally `npm install <tarbal>`, but without success. To be honest I don't know how to test. The build works fine though.

Any help or ideas on that topic are welcome.

### CODE REVIEW

**Comment by doliG** [comment]
I have the same commitlint error as #413 .
IMHO it's a bug on circle CI side 🤔

**Comment by guastallaigor** [comment]
I see, I will try to debug it today or tomorrow the commitlint CI bug.
But I have to say I don't have much knowledge about it, it was another member that configured Circle CI.

**Comment by doliG** [comment]
Ok thank you and good luck 🙏

**Comment by trezy** [comment]
I can take a look at that. I dealt with it recently on another repo. 😉

**Comment by doliG** [comment]
Hello @trezy , any updates on the CI bug ?

**Comment by virtuoushub** [comment]
tl;dr _to fix the failed commitlint check, just [retry in CircleCI](https://circleci.com/gh/nostalgic-css/NES.css/3918?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)._

@trezy / @BcRikko or someone else who has admin to this project's CircleCI; I think the failure on CircleCI has little to do with this PR code, and more to do with a non deterministic build via the combination of this code's CircleCI config with williamlauze/circleci-commitlint ( [github](https://github.com/wilau2/circleci-commitlint-step) / [dockerhub](https://hub.docker.com/r/williamlauze/circleci-commitlint) ).

Notice that PR #413 also failed the same way.

I wired up CircleCI in my fork and was unable to reproduce the CircleCI build failure for either this PR or #413. I recommend retrying the build to see if that fixes it short term.

Longer term, see if you can isolate why the williamlauze/circleci-commitlint dependency seems to behave non-deterministically with your repo. See also https://github.com/wilau2/circleci-commitlint-step/issues/13

cc @doliG @guastallaigor

**Comment by guastallaigor** [comment]
It's very strange what is happening here. I've tried to debug in another repo that I use the commitlint, which is [vue-horizontal-timeline](https://github.com/guastallaigor/vue-horizontal-timeline), you can see the workflow [here](https://circleci.com/gh/guastallaigor/vue-horizontal-timeline/235) that is currently does not work, but it also doesn't fail.

Also I've tried in this repo to make another PR to see what it was going to happen. Turns out it happened the same thing, but I couldn't replicate your scenario.

image

As of this moment, I wasn't able to fix the error, possibly due to a bug inside the commitlint script which is used inside Circle CI. So if any of us can fix the issue, the other options as I see it is:

1. Change to another one like [this](https://github.com/fathyb/commitlint-circle)
2. Remove the commitlint step for now, until the apparently bug is fixed

**Comment by trezy** [comment]
Just letting everybody know that I haven’t forgotten about this. I’ve got several projects that are breaking because CircleCI has made changes that have broken the Commitlint Docker image. There are a couple of solutions, I’ve just gotta get a working example of one that will work for us.

**Comment by ldd** [comment]
I was able to find a similar issue here:
https://github.com/conventional-changelog/commitlint/issues/6

Can anyone comment on whether that fixes things or not?

**Comment by virtuoushub** [comment]
@ldd the issue w/ the commitlint CircleCI job is due to a bug in https://github.com/wilau2/circleci-commitlint-step

I opened a PR w/ the fix ( https://github.com/wilau2/circleci-commitlint-step/pull/18 )

Since that is an external dependency and there is no telling if/when the PR will get accepted and the docker image built/pushed up for this project to use, I recommend we go w/ @guastallaigor's suggestion of removing the commitlint step for now.

It's failure is currently blocking:

https://github.com/nostalgic-css/NES.css/pull/413
https://github.com/nostalgic-css/NES.css/pull/417
https://github.com/nostalgic-css/NES.css/pull/421
https://github.com/nostalgic-css/NES.css/pull/422

P.S.
@guastallaigor I think there is something weird w/ CircleCI and that bash script where it's exit code is not always properly captured, so sometimes it reports as a success in CircleCI and sometimes it doesn't, as illustrated by your attempt to reproduce.
